src/datepicker/adapters/ngb-date-native-utc-adapter.ts

```diff
@@ -14,6 +15,16 @@ export class NgbDateNativeUTCAdapter extends NgbDateAdapter<Date> {
   }
 
   toModel(date: NgbDateStruct): Date {
-    return date && date.year && date.month ? new Date(Date.UTC(date.year, date.month - 1, date.day)) : null;
+    if (date && isNumber(date.year) && isNumber(date.month) && isNumber(date.day)) {
```

Comment:
There is quite a bit of code duplication between adapters - could you think of a way to remove it?